### PR TITLE
Update creating-postgresql-service-containers.md

### PR DESCRIPTION
**Pull request by nyvelius**
### Why:

Minor issue in readme.

### What's being changed:

Fix minor issue in readme.

### Check off the following:

- [X] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [X] For content changes, I have reviewed the [Content style guide for GitHub Docs]

### CODE REVIEW

**Comment by janiceilene** [comment]
@nyvelius Thanks so much for opening a PR! 

Can you clarify why you think the language in this article should be updated? Is it currently incorrect or do you think the suggested wording is clearer in some way? That information will help me get this into the hands of the right reviewers ⚡

**Comment by nyvelius** [comment]
Sure. The original version refers to creating a _client_, when what I think is meant, is a _database_.

**Review by skedwards88 — COMMENTED** [review]
💖 Thank you for opening this PR! For consistency, we would want these changes applied across the file since there are multiple places where this script is referenced. (This text is referring to the `client.js` script, which is defined at the end of the file.) Are you willing to make those additions?

**Comment by martin389** [comment]
### Changes in this PR
| Article in staging  | Changes |
| --- | --- | 
“[Creating PostgreSQL service containers](https://docs-5391--patch-1.herokuapp.com/en/actions/guides/creating-postgresql-service-containers)” | Revises intro text

### Versioning
dotcom, GHES 2.22 and higher

### Tests
All tests are passing! :green_apple:

File: content/actions/guides/creating-postgresql-service-containers.md [hunk 1]
```diff
@@ -324,11 +324,11 @@ pgclient.query('SELECT * FROM student', (err, res) => {
 });
 ```
 
-The script creates a new PostgreSQL `Client`, which accepts a `host` and `port` parameter. The script uses the `POSTGRES_HOST` and `POSTGRES_PORT` environment variables to set the client's IP address and port. If `host` and `port` are not defined, the default host is `localhost` and the default port is 5432.
+The script creates a new connection to the PostgreSQL service, and uses the `POSTGRES_HOST` and `POSTGRES_PORT` environment variables to define the PostgreSQL service IP address and port. If `host` and `port` are not defined, the default host is `localhost` and the default port is 5432.
```
Comment (lucascosti) [inline content/actions/guides/creating-postgresql-service-containers.md:327]: Would specify be more correct? The IP address and port already exists, we're just specifying them for the client.

```suggestion
The script creates a new connection to the PostgreSQL service, and uses the `POSTGRES_HOST` and `POSTGRES_PORT` environment variables to specify the PostgreSQL service IP address and port. If `host` and `port` are not defined, the default host is `localhost` and the default port is 5432.
```

**Review by lucascosti — DISMISSED** [review]
Looks good! just a few niggly things 🙂

**Review by lucascosti — APPROVED** [review]
⚡

**Comment by erickadalberto** [comment]
#6342

**Pull request merged**